src/models/message_view.ts

```diff
@@ -0,0 +1,84 @@
+import * as t from "io-ts";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { Container } from "@azure/cosmos";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
+import { FiscalCode } from "../../generated/definitions/FiscalCode";
+import { MessageStatusValue } from "../../generated/definitions/MessageStatusValue";
+import { ServiceId } from "../../generated/definitions/ServiceId";
+import { TimeToLiveSeconds } from "../../generated/definitions/TimeToLiveSeconds";
+import { Timestamp } from "../../generated/definitions/Timestamp";
+import { CosmosResource, CosmosdbModel } from "../utils/cosmosdb_model";
+
+export const MESSAGE_VIEW_COLLECTION_NAME = "message-view";
+const MESSAGE_VIEW_MODEL_PK_FIELD = "fiscalCode";
+
+export const Component = t.interface({
```

Comment:
This component is used to mark the existing of specific message suparts (eucovidcert, attachments, etc...).
The Component type is the base type for a message subpart: it could be extended to add other specific info required only by a specific subpart (eg the rptId for Payment).